statsmodels/discrete/discrete_model.py

```diff
@@ -3214,11 +3206,6 @@ def fit(self, start_params=None, method='bfgs', maxiter=35,
 
         nbinfit = NegativeBinomialResults(self, mlefit._results)
         result = NegativeBinomialResultsWrapper(nbinfit)
-
-        if cov_kwds is None:
-            cov_kwds = {}
-        result._get_robustcov_results(cov_type=cov_type,
-                                    use_self=True, use_t=use_t, **cov_kwds)
```

Comment:
This isn’t quite right.  Use_t and cov_type need to be passed to the super.fit call.  The fact that failing to do this apparently doesn’t break any tests means I need to add some tests.

Reply:
Fixed.